src/components/views/elements/AccessibleTooltipButton.tsx

```diff
@@ -62,7 +62,8 @@ export default class AccessibleTooltipButton extends React.PureComponent<IToolti
     };
 
     render() {
-        const {title, tooltip, children, tooltipClassName, ...props} = this.props;
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        const {title, tooltip, children, tooltipClassName, forceHide, ...props} = this.props;
```

Comment:
Seems unused...?

Reply:
It is, hence the comment above, without this it gets left in `...props` and then passed to a native react element which it complains as `forceHide` is not known to a Div

Reply:
Ah ah, okay.